usd_qtpy/render_util/framing_camera.py

```diff
@@ -0,0 +1,270 @@
+# Generates Stage framing camera.
+# heavy inspiration taken from: 
+# https://github.com/beersandrew/assets/tree/1df93f4da686e9040093027df1e42ff9ea694866/scripts/thumbnail-generator
+
+import math
+from typing import Union
+
+from pxr import Usd, UsdGeom
+from pxr import Sdf, Gf
+
+def _stage_up(stage: Usd.Stage) -> str:
+    return UsdGeom.GetStageUpAxis(stage)
+
+def create_framing_camera_in_stage(stage: Usd.Stage, root: Union[Sdf.Path,str], 
+                                   name: str = "framingCam", fit: float = 1,
+                                   width: int = 16, height: int = 9, for_turntable: bool = False) -> UsdGeom.Camera:
+    """
+    Adds a camera that frames the whole stage.
+    Can be specified to have a different aspect ratio, this will affect the sensor size internally.
+    """
+    if isinstance(root,str):
+        root = Sdf.Path(root)
+    # create camera
+    camera = create_perspective_camera_in_stage(stage, root, name, width, height)
+
+    # Do prerequisite math so that functions don't have to run these same operations.
+
+    bounds = get_stage_boundingbox(stage)
+    bounds_min, bounds_max = bounds.GetMin(), bounds.GetMax()
+
+    centroid = (bounds_min + bounds_max) / 2
+
+    z_up = (_stage_up(stage) == "Z")
```

Comment:
Redundant parentheses
```suggestion
    z_up = _stage_up(stage) == "Z"
```

Reply:
Adjusted. I always get a bit paranoid about bools resolving properly, I'll try to be more courageous in my endeavours